### PR TITLE
Adds new integration [rytilahti/homeassistant-upnp-availability]

### PR DESCRIPTION
**Pull request by rytilahti**
**Link to successful HACS action:**  https://github.com/rytilahti/homeassistant-upnp-availability/actions/runs/3552256009/jobs/5967049683
**Link to successful hassfest action (if integration):**  https://github.com/rytilahti/homeassistant-upnp-availability/actions/runs/3552256009/jobs/5967049683

Both actions are run in a single flow, therefore a duplicate link above :-)

Closes https://github.com/rytilahti/homeassistant-upnp-availability/issues/10

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -656,6 +656,7 @@
   "ryanmac8/Home-Assistant-Marta",
   "ryannazaretian/hacs-nexia-climate-integration",
   "ryanwinter/hass-rainforest-emu-2",
+  "rytilahti/homeassistant-upnp-availability",
```
Comment (ludeeus) [inline integration:659]: https://github.com/hacs/default/actions/runs/3724471340/jobs/6316646938#step:3:18 CI is failing

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 1 above)
Comment (rytilahti) [reply integration:659]: Indeed, the `ignore: brands` for the hacs action made the checks only pass on the repository itself, so this is pending for the merging of https://github.com/home-assistant/brands/pull/3907

Comment (rytilahti) [reply integration:685]: https://github.com/home-assistant/brands/pull/3907 is now merged so the CI should pass *fingers crossed*.

**Comment by rytilahti** [comment]
Thanks @ludeeus! :tada:

**Pull request merged**